src/Question.js

```diff
@@ -10,19 +10,32 @@ class Question extends Component {
       option: PropTypes.string.isRequired,
       correct: PropTypes.bool.isRequired
     })),
-    onClick: PropTypes.func.isRequired
+    onClick: PropTypes.func.isRequired,
+    questionNumber: PropTypes.number.isRequired,
+    userHasAnswered: PropTypes.bool.isRequired
+  }
+  getQuestionClassName() {
+    const questionClassName = {
+      correct ? 'answered' : 'wrong-answer'
```

Comment:
Like I said in our chat, I tried a number (somewhere in the double digits) of variations on correct such as this.props.answer.correct, this.answer.correct, this.props.correct, const itsright = {this.props.answer.correct} (itsright? 'answered...), itsright = {answer.correct} ... 

I had something similar in my earlier handleClick that was able to accurately acknowledge whether the answer is correct. 

I don't really know what you're saying at the end of this comment though, and don't feel like this clears anything up :/